codegen/smithy-go-codegen/src/main/resources/software/amazon/smithy/go/codegen/smithy-tests/int-enum-shape-test/expected/types/enums.go

```diff
@@ -0,0 +1,66 @@
+// Code generated by smithy-go-codegen DO NOT EDIT.
```

Comment:
Not relevant to this PR specifically, but the `codegen` directory should be broken off into an empty go module so any go files nested under codegen are out of scope of go commands executed on the smithy-go root module.